Core/QueuePlayer/QueuePlayer.swift

```diff
@@ -75,6 +81,7 @@ public class QueuePlayer {
             oldValue?.actions = nil
         }
     }
+    private var playbackRate: Float = 1.0
```

Comment:
Since the rate is persisted and also audio player stores it, I don't think we need this property.